website/static/js/logTextParser.js

```diff
@@ -220,25 +220,37 @@ var LogPieces = {
         view: function (ctrl, logObject) {
             var contributors = logObject.attributes.params.contributors;
             if(paramIsReturned(contributors, logObject)) {
-                return m('span', contributors.map(function(item, index, arr){
-                    var comma = ' ';
-                    if(index !== arr.length - 1){
-                        comma = ', ';
-                    }
-                    if(index === arr.length-2){
-                        comma = ' and ';
-                    }
+                return m('span', (function (){
+                    var contribList = [];
+                    var numContribShown = 15;
```

Comment:
Also, though this was vague and unspecified in the ticket, let's make the `numContribShown` 3 rather than 15.